### PR TITLE
Mobile registration website

### PR DESCRIPTION
**Pull request by omartehsin1**
# Summary | Résumé

UI updates to some of the pages to make them more mobile-friendly:
 - Expanded some buttons to make them easier to press when on mobile
 - adjusted Start page image to appear above the text
 - Added "Top of Page" button for mobile users to navigate to the top of the page
 - Additional styling and padding

TO BE COMPLETED:

FR translation for "Top of Page"

Link: https://covid-alert-mobile-regi-ivmcvd.herokuapp.com/en/

### CODE REVIEW

**Comment by dsamojlenko** [comment]
Minor diff in the Beta banner (vertical alignment of Beta label)

Design | Actual
------- | -------
image | image

**Comment by dsamojlenko** [comment]
Missing the "Top of page" link in footer

image

**Comment by dsamojlenko** [comment]
This looks great! Just a few minor changes.

File: register/templates/register/includes/footer.html [hunk 1]
```diff
@@ -10,6 +10,10 @@
                 <li><a href="">{% trans "Terms of use" %}</a></li>
             </ul>
         </div>
+        <div class="box-style">
+            <a href="" class="button-text-style">{% trans "Top of Page" %}</a>
```
Comment (dsamojlenko) [inline register/templates/register/includes/footer.html:14]: you should use `<a href="#page--content"` to return to top without reloading the page

Comment (omartehsin1) [reply register/templates/register/includes/footer.html]: updated

**Review by dsamojlenko — APPROVED** [review]
Looks good!

**Pull request merged**